src/controllers/comments/deleteCommentById.js

```diff
@@ -0,0 +1,18 @@
+const { deleteCommentByIdQuery } = require('../../database/queries');
+
+const deleteCommentById = (req, res, next) => {
+  const { id } = req.params;
+  deleteCommentByIdQuery(id)
+    .then(() => {
+      res.status(200).json({
+        error: false,
+        data: {
+          message: 'Comment Deleted successfully',
+        },
+      });
+    })
+    .then(() => res.redirect('/post'))
```

Comment:
Consider removing the `res.redirect('/post')` line from the code as it seems to be an unnecessary redirection after the comment deletion.